mathesar_ui/src/stores/tables.ts

```diff
@@ -120,6 +121,13 @@ export function getTablesStoreForSchema(schemaId: SchemaEntry['id']): Writable<D
   return store;
 }
 
+export function createTable(
+  schema: SchemaEntry['id'],
+  name: string | undefined = undefined,
```

Comment:
An alternate representation could be:

```suggestion
  name?: string,
```

I personally find this cleaner.

We do not have to make any changes for this. I only want to understand if you had reasons for explicitly mentioning `undefined`.

Reply:
Thanks for pointing that out. I prefer that as well and have opened #633 to implement that change. My head has been in Python and mypy for the past couple months so it just didn't occur to me.